example/bufferCopy/src/bufferCopy.cpp

```diff
@@ -283,6 +283,10 @@ auto main()
     alpaka::mem::view::copy(devQueue, deviceBuffer1, hostViewPlainPtr, extents);
     alpaka::mem::view::copy(devQueue, deviceBuffer2, hostBuffer, extents);
 
+    // Depending on the accelerator, the allocation function may introduce
+    // padding between rows/planes of multidimensional memory allocations.
+    // Therefore the pitch (distance between consecutive rows/planes) may be
```

Comment:
This might be slightly confusing for readers now, since the pitch is always greater or equal the data size. Also, both padding and pitch are introduced as terms, but not brought into correlation to each other. Also, the "why" is not yet clear for new users.

A better wording might be:

Depending on the accelerator, the allocation function may introduce padding between rows/planes of multidimensional memory allocations to avoid bank conflicts during data access.
The data size plus this padding is called the pitch and is equivalent to the distance between consecutive rows/planes.
Consequently, pitched data with non-zero padding occupies (slightly) more memory than non-pitched data.